rpm/pacemaker.spec.in

```diff
@@ -338,7 +338,7 @@ BuildRequires: %{pkgname_glue_libs}-devel
 %if %{with doc}
 BuildRequires: asciidoc
 BuildRequires: inkscape
-BuildRequires: %{python_name}-sphinx
+BuildRequires: %{python_name}-sphinx >= 3.1.0
```

Comment:
This was released in June 2020, so most of our supported distros won't have it. Is there a way around this?

Reply:
Ah, bummer.  Yeah - I can do everything the long way by hand instead.